ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -97,13 +34,34 @@ class TooltipView: UIView {
 
         messageLabel.text = message
         messageLabel.textAlignment = textAlignment
-        messageLabel.isAccessibilityElement = false
-        addSubview(messageLabel)
+
+        if let titleLabel = titleLabel, let title = title {
+            titleLabel.text = title
+            titleLabel.textAlignment = textAlignment
+        }
+
+        addSubview(textContainer)
 
         // Shadow
-        layer.shadowOpacity = Constants.shadowOpacity
-        layer.shadowRadius = Constants.shadowRadius
-        layer.shadowOffset = CGSize(width: 0.0, height: Constants.shadowOffset)
+        let backgroundCornerRadius = tokenSet[.backgroundCornerRadius].float
+        let shadowInfo = tokenSet[.shadowInfo].shadowInfo
+        let ambientShadow = CALayer()
```

Comment:
this this code different from updateShadows() function you just created?

Reply:
this adds the shadows (done once in the init) and the new method updates the existing shadows (readding the shadows makes the shadow opacity get darker and darker with the added layers)

Reply:
with bit of the refactor, it seems like the code can be reused ?